meta/recipes-devtools/gcc/gcc-target.inc

```diff
@@ -169,6 +173,7 @@ do_install () {
 
 	# We don't care about the gcc-<version> ones for this
 	rm -f *gcc-?.?*
+	rm -f *gcc-??.?*
```

Comment:
Is this an upstream bug?

Reply:
If you care about the space upstream, then yes

Reply:
OK. Well, if you had the luxury of time, I would suggest that you submit it upstream. But that isn't blocking for this review.